engines/python/setup/djl_python/output_formatter.py

```diff
@@ -191,14 +213,50 @@ def _jsonlines_chat_output_formatter(token: Token, first_token: bool,
     return json_encoded_str
 
 
-def sse_response_formatter(*args, **kwargs):
+def sse_response_formatter(request_output: RequestOutput):
     """
     Decorator that used to form as SSE
     """
-    output_str = _jsonlines_output_formatter(*args, **kwargs)
+    output_str = _jsonlines_output_formatter(request_output)
     return f"data: {output_str}\n"
 
 
+def adapt_legacy_output_formatter(request_output: TextGenerationOutput):
```

Comment:
comments to add a `@deprecate` flag. Plan to remove in 0.29.0

Reply:
Done